src/pages/waiting-room-to-car/cat-be/components/vehicle-checks-modal/vehicle-checks-modal.cat-be.page.html

```diff
@@ -35,4 +32,5 @@
       <p>Some tell me stuff!!</p>
     </ion-card-content>
   </ion-card>
-</ion-content>
+  <warning-banner warningText="This will result in 4 driving faults and 1 serious fault" *ngIf="((pageState.vehicleChecksScore$ | async).drivingFaults === 4) && ((pageState.vehicleChecksScore$ | async).seriousFaults === 1)"></warning-banner>
```

Comment:
any way we can bring this into the component so it can be unit tested?

Reply:
did it